src/main/scala/com/fulcrumgenomics/vcf/HapCutToVcf.scala

```diff
@@ -117,6 +123,32 @@ class HapCutToVcf
     vcfWriter.close()
   }
 
+  /** If fixing ambiguous (IUPAC) bases is desired, returns a new path to a VCF, otherwise returns the original input VCF. */
+  private def inputVcf: FilePath = if (!fixAmbiguousReferenceAlleles) this.vcf else {
```

Comment:
Could you move this above `execute()` please?

Reply:
Done